easybuild/easyconfigs/b/BCFtools/BCFtools-1.9-foss-2018b.eb

```diff
@@ -0,0 +1,33 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'ConfigureMake'
+
+name = 'BCFtools'
+version = '1.9'
+
+homepage = 'http://www.htslib.org/'
+description = """Samtools is a suite of programs for interacting with high-throughput sequencing data.
+ BCFtools - Reading/writing BCF2/VCF/gVCF files and calling/filtering/summarising SNP and short indel sequence
+ variants"""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/samtools/%(namelower)s/releases/download/%(version)s']
+sources = [SOURCELOWER_TAR_BZ2]
+checksums = ['6f36d0e6f16ec4acf88649fb1565d443acf0ba40f25a9afd87f14d14d13070c8']
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('HTSlib', '1.9'),
+    ('GSL', '2.5', '-GCCcore-7.3.0', True),
```

Comment:
@fizwit No need to hardcode to `GCCcore` here (that's also what's breaking the tests):

```python
('GSL', '2.5'),
```

Reply:
Easybuild should resolve lower toolchains, but sometimes it does not work.
EasyBuild/3.6.2
ERROR: Build of /home/neo/BCFtools-1.9-foss-2018b.eb failed (err: 'build failed (first 300 chars): Missing modulesfor one or more dependencies: GSL/2.5-foss-2018b')
outside_user@1b732d5d33d1:/home/neo$ ml avail GSL

------------------------------------------------ /app/modules/all -------------------------------------------------
   GSL/2.3-foss-2016b    GSL/2.5-GCCcore-7.3.0 (D)

Reply:
@fizwit This is related to a known problem, see https://github.com/easybuilders/easybuild-framework/issues/2375 .

The dependency resolution mechanism currently requires that the `GSL-2.5-GCCcore-7.3.0.eb` easyconfig file is included in the robot search path; if it's not there, only `GSL/2.5-foss-2018b` (i.e. the top-level toolchain) is considered.

I hope to find some time to try and fix that bug soon...